pkg/reconciler/v1alpha1/taskrun/resources/pod_test.go

```diff
@@ -510,3 +484,165 @@ func TestMakeAnnotations(t *testing.T) {
 		})
 	}
 }
+
+func TestInitOutputResourcesDefaultDir(t *testing.T) {
+	names.TestingSeed()
+
+	randReader = strings.NewReader(strings.Repeat("a", 10000))
+	defer func() { randReader = rand.Reader }()
+
+	for _, c := range []struct {
+		desc         string
+		trs          v1alpha1.TaskRunSpec
+		ts           v1alpha1.TaskSpec
+		bAnnotations map[string]string
+		want         *corev1.PodSpec
+		wantErr      error
+	}{{
+		desc: "task-with-output-image-resource",
+		trs: v1alpha1.TaskRunSpec{
+			Outputs: v1alpha1.TaskRunOutputs{
+				Resources: []v1alpha1.TaskResourceBinding{{
+					Name: "outputimage",
+					ResourceRef: v1alpha1.PipelineResourceRef{
+						Name: "outputimage",
+					},
+				}},
+				Params: []v1alpha1.Param{},
```

Comment:
i think this might be causing a problem now b/c of https://github.com/tektoncd/pipeline/commit/93de2fcc6d282242ed664613bb385f3596f8998e :(